docs/unified-test-documentation/dasharo-compatibility/31L-smbios.md

```diff
@@ -0,0 +1,392 @@
+# Dasharo Compatibility: SMBIOS
+
+## Test cases
+### DMI001.001 Verify the serial number
+
+**Test description**
+
+This test aims to verify that the serial number field is filled in correctly
+according to the [Dasharo SMBIOS guidelines](https://docs.dasharo.com/dev-proc/smbios-rules/)
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+1. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+1. Install the `dmidecode` package: `sudo apt install dmidecode`
+1. Obtain the stock firmware dump from [cloud.3mdeb.com](https://cloud.3mdeb.com/index.php/f/391118).
```

Comment:
This is information leak, we cannot expose publicly binaries dumped from the machines since those can contain sensitive information. Luckily this is internal link, but it just mean providing that is useless for external entities - so somehow should be either explained either changed.